layouts/partials/blocks/awards.html

```diff
@@ -0,0 +1,222 @@
+                    header: ".state-awards-header",
+                    container: ".state-awards",
+                  },
+                  {
+                    header: ".state-finalists-header",
+                    container: ".state-finalists",
+                  },
+                ];
+
+                sections.forEach(({ header, container }) => {
+                  const section = awardsContainer.querySelector(container);
+                  const headerElement = awardsContainer.querySelector(header);
+
+                  const hasVisibleAwards = Array.from(
+                    section.querySelectorAll(".award-entity"),
+                  ).some((award) => award.style.display === "block");
+                  headerElement.style.display = hasVisibleAwards
+                    ? "block"
+                    : "none";
+                });
+              });
+            });
+
+            yearItems[0]?.click();
+          });
+        </script>
+
+        <div
+          class="flex w-full flex-wrap pb-[15px] pl-[15px] pr-0 pt-[15px] sm:pl-[25px] sm:pr-[20px]"
+        >
+          <div
+            class="awards-container"
+            style="flex: 1 1 100%; word-break: break-word;"
+          >
+            <p
+              class="award-national-state national-awards-header pb-[10px] text-[22px] font-bold"
+            >
+              National Awards
+            </p>
+            <div class="national-awards">
+              {{ range where (where . "national_or_state" "National") "winner_or_finalist_honorable_mention" "Winner" }}
+                <div class="award-entity" data-year="{{ .year }}">
+                  {{ .entity }}:
+                  <span class="award-category font-bold"
+                    >{{ .award_category }}</span
+                  >
+
+                  <p class="award-story mb-5">
+                    {{ if .link }}
+                      {{ partial "tag/maybe-link.html" (dict
+                        "link" .link
+                        "text" .story
+                        "class" "award-story-link text-[#22416e] underline transition-colors duration-500 ease-linear hover:text-[#ffcb03]"
+                        )
+                      }}
+                    {{ else }}
+                      {{ .story }}
+                    {{ end }}
+                  </p>
+                </div>
+              {{ end }}
+            </div>
+
+            <p
+              class="award-national-state national-finalists-header pb-[10px] text-[22px] font-bold"
+            >
+              National Finalists
+            </p>
+            <div class="national-finalists">
+              {{ range where (where . "national_or_state" "National") "winner_or_finalist_honorable_mention" "Finalist/Honorable Mention" }}
+                <div class="award-entity" data-year="{{ .year }}">
+                  {{ .entity }}:
+                  <span class="award-category font-bold"
+                    >{{ .award_category }}</span
```

Comment:
I find this wrapping ugly, so usually I do

```
<span class="award-category font-bold">
    {{- .award_category -}}
</span>
```

The `{{-` and `-}}` trim the whitespace between the tag and template code. The usually only matters for links where if you don't do it, the underline is too long because it's underlining a space.